OneSignalSDK/unittest/src/test/java/com/test/onesignal/RemoteParamsTests.java

```diff
@@ -174,7 +172,9 @@ public void testUserPrivacyConsentRequired_UserConfigurationNotOverrideRemotePar
 
     @Test
     public void testUserPrivacyConsentNotRequired_UserConfigurationNotOverrideRemoteParams() throws Exception {
-        ShadowOneSignalRestClient.setRemoteParamsGetHtmlResponse();
+        JSONObject remoteParams = new JSONObject();
+        remoteParams.put("requires_user_privacy_consent", false);
+        ShadowOneSignalRestClient.setRemoteParamsGetHtmlResponse(remoteParams);
```

Comment:
change to ` ShadowOneSignalRestClient.setRemoteParamsRequirePrivacyConsent(false);`

Reply:
ah my bad, I changed two instances but there was other pre-existing code I copied this from.
I am just going to update them all in another commit in this PR.

Reply:
@Jeasmine Fixed in commit 8a6b8828738abe803f38d6014d6969ca8a6c4753